release/staging-resources-cdk/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "staging-resources-cdk",
+  "version": "0.1.0",
+  "bin": {
+    "staging-resources-cdk": "bin/staging-resources-cdk.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "test": "jest",
```

Comment:
If I recall correctly, there is a part of the CDK output that can be very large. Can we introduce a clean step as well?